### PR TITLE
Feature/44 page list

### PR DESCRIPTION
**Pull request by sdorra**
Implemented page list.

### CODE REVIEW

File: src/main/js/wiki/containers/Directory.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    const { repository, branch } = ownProps.match.params;
+    const path = findDirectoryPath(ownProps);
+    const url = createDirectoryUrl(repository, branch, path);
+
+    const wikiId = createId(repository, branch);
+    const wiki = state.wiki[wikiId] ||{};
+
+
+    let baseDirectory = '';
+    if (wiki.wiki && wiki.wiki.directory) {
```
Comment (mbehlendorf) [inline src/main/js/wiki/containers/Directory.js]: Rename wiki

File: src/main/js/wiki/containers/Directory.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    fetchWikiIfNeeded: (repository: string, branch: string) => void,
+    fetchDirectoryIfNeeded: (url: string) => void
+}
+
+class Directory extends React.Component<Props> {
+
+    componentDidMount() {
+        const { url, repository, branch } = this.props;
+
+        this.props.fetchDirectoryIfNeeded(url);
```
Comment (mbehlendorf) [inline src/main/js/wiki/containers/Directory.js]: You could bind the function like the other variables

File: src/main/js/wiki/containers/Page.js [hunk 3]
```diff
@@ -124,8 +126,18 @@ const mapStateToProps = (state, ownProps) => {
     const wikiId = createId(repository, branch);
     const wiki = state.wiki[wikiId] ||{};
 
+    let pagesLink = '#';
+    if (wiki.wiki && wiki.wiki.directory) {
```
Comment (mbehlendorf) [inline src/main/js/wiki/containers/Page.js]: Rename wiki

**Comment by sdorra** [comment]
Was reopened by the stage1

**Pull request closed without merging**